examples/finite_prime_field.rs

```diff
@@ -0,0 +1,254 @@
+use noether::{
+    AssociativeAddition, AssociativeMultiplication, CommutativeAddition, CommutativeMultiplication,
+    Distributive, FiniteField,
+};
+use num_traits::{Euclid, Inv, One, Zero};
+use std::ops::{Add, AddAssign, Div, DivAssign, Mul, MulAssign, Neg, Rem, Sub, SubAssign};
+
+/// The trait hierarchy in Noether is useful for verifying the correct and idiomatic implementation
+/// of algebraic structures in rust.
+///
+/// We take here, as an example, the skeleton implementation of a finite field.
+/// Where `L` number of words and `D` number of words are used to represent the field.
+/// The modulus is the order of the field, which would be prime.
```

Comment:
Or a prime power, unless we don't care about these?

Reply:
Maybe later, I could add that it could be either.

Reply:
Added some additional detail here.